tests/e2e/specs/shopper/checkout-different-addresses.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	uiUnblocked,
+} from '@woocommerce/e2e-utils';
+
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+
+import { BILLING_DETAILS, SHIPPING_DETAILS } from '../../../utils/constants';
+const SIMPLE_PRODUCT_NAME = '128GB USB Stick';
+
+const config = require( 'config' );
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Checkout tests', () => {} );
+
+describe( 'Shopper → Checkout → Can have different shipping and billing addresses', () => {
+	beforeAll( async () => {
+		// prevent CartCheckoutCompatibilityNotice from appearing
+		await preventCompatibilityNotice();
+		await merchant.login();
+
+		// Activate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
+		await saveOrPublish();
+	} );
+
+	afterAll( async () => {
+		// Deactivate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
```

Comment:
The same thing as in the `beforeAll` block! The **"Company"** field is not being disabled here. This time let's use the `unsetCheckbox` Woo's utility function:

```js
/**
 * External dependencies
 */
import { unsetCheckbox } from '@woocommerce/e2e-utils';

// Deactivate the "Company" field
await unsetCheckbox('#inspector-toggle-control-1');
```

And no need to deactivate the remaining fields since they were active by default.

Reply:
Changed! 😀